addons/website_sale/views/templates.xml

```diff
@@ -1516,7 +1516,7 @@
                 <span class=''>
                     <t t-set='force_coupon' t-value="website_sale_order.pricelist_id.code or request.params.get('code_not_available')"/>
                     <t t-if="not force_coupon">
-                        <a href="#" class="show_coupon">I have a promo code</a>
```

Comment:
The problem comes from the parent <td/> which have no padding due to `p-0`, I think removing the class solves this problem.